CHANGELOG.md

```diff
@@ -1,9 +1,27 @@
 # Change Log
 
+## Post-3.0.1 (2025-04-24)
+
+### Changes since 3.0.1
+
+Items marked 'Removed,' 'Fixed,' or 'Added' indicate potential
```

Comment:
Does the 'Added' affect compatibility?  Agree with 'Removed' and 'Fixed'.

Reply:
Agree. I have updated the notes.